backend/src/routes/api.ts

```diff
@@ -0,0 +1,75 @@
+import { pick } from "lodash";
+import { NextFunction, Request, Response, Router } from "express";
+import { isValidObjectId } from "mongoose";
+import NotFoundError from "../controllers/errors/NotFoundError";
+import TechController from "../controllers/TechController";
+import logger from "../logger";
+import Tech from "../models/Tech";
+import "../types/express";
+
+/* Dependencies */
+
+const techController = new TechController(Tech);
+
+/* API routes */
+
+const api = Router();
+
+api.use((err: Error, req: Request, res: Response, next: NextFunction) => {
+  if (err instanceof NotFoundError) {
+    res.status(404).json({ errors: ["id_not_found"] });
+  } else {
+    logger.error("Unrecognized error " + err.message, err);
+    res.status(500).json({ errors: ["internal_server_error"] });
+  }
+});
+
+api.get("/v1/techs/:id", async (req, res) => {
+  const tech = await techController.getById(req.params["id"]);
+  res.json(tech);
+});
+
+api.post(
+  "/v1/techs/:id",
+  (req, res, next) => {
+    if (req.user) {
+      next();
+    } else {
+      res.status(401).json({ errors: ["unauthenticated"] });
+    }
+  },
+  (req, res, next) => {
+    if (req.user && req.user.isAdmin) {
```

Comment:
Note: most of the imageUrl fields are null at the moment in the database, but this endpoint allows the image URLs to be updated in production.

This middleware will check if the user trying to make a change to the data record is an admin or not